src/itemlistformaction.cpp

```diff
@@ -1019,6 +1019,7 @@ std::string ItemListFormAction::item2formatted_line(const ItemPtrPosPair& item,
 	FmtStrFormatter fmt;
 	fmt.register_fmt('i', strprintf::fmt("%u", item.second + 1));
 	fmt.register_fmt('f', gen_flags(item.first));
```

Comment:
@Minoru: Do you think it is useful to add the other flags (normally part of `f` with [gen_flags()](https://github.com/newsboat/newsboat/blob/09ec32244dca3f5699cb736636c8b5a2105d31ef/src/itemlistformaction.cpp#L1374-L1390)) as well? (i.e. a format specifier for `item->deleted()` and a format specifier for `item->flags().length()`)